.yarnrc.txt

```diff
@@ -0,0 +1 @@
+network-timeout 600000
```

Comment:
In CI on Windows machines, sometimes `yarn install` keeps timing out and restarting. In `lwc-dev-mobile` we used this `.yarnrc` file to work around that so adding it here too.